be/src/exec/sorting/merge_column.cpp

```diff
@@ -340,7 +340,7 @@ ChunkUniquePtr SortedRun::clone_slice() const {
     }
 }
 
-ChunkPtr SortedRun::steal_chunk(size_t size, size_t skipped_rows) {
+std::pair<ChunkPtr, Columns> SortedRun::steal_chunk(bool steal_orderby, size_t size, size_t skipped_rows) {
```

Comment:
how about add a new function?

Reply:
I've thought about that, but the new function has the similar logic of this one. And finally I choose to add an condition.😢

Reply:
Updated. add a new function `steal` which returns both. and `steal_chunk` will delegate to `steal`